packages/playground/src/utils/deploy_vm.ts

```diff
@@ -134,6 +145,10 @@ export interface Machine {
   disks?: Disk[];
   country?: string;
   qsfsDisks?: QsfsDisk[];
+  hasGPU?: boolean;
+  rentedBy?: number;
+  nodeId?: number;
```

Comment:
Based on the script in the client https://github.com/threefoldtech/tfgrid-sdk-ts/pull/666/files#diff-75c77c88440bd55706ceb5a41cccb3f087c0b5bb3645fb6cb8435b5576b70b04R24-R26
I have to pass these attrs, and these attrs came from the `full_vm` component, I added them to the machine interface because I need them there.